shapely/libgeos.py

```diff
@@ -0,0 +1,116 @@
+"""
+Proxies for the GEOS shared library, versions, and configure options
+TODO: find a flexible way to load a dynamic library from a specified path
```

Comment:
Should this be set from a `GEOS_LIBRARY_PATH` environment variable, as shown in #240?